### PR TITLE
Refined initial text.

### PR DESCRIPTION
**Pull request by hjoliver**
Some minor fixes and refinements in the introductory sections.

### CODE REVIEW

File: cylc-overview.tex [hunk 1]
```diff
@@ -5,21 +5,20 @@ \section{Cylc Overview}
 Cylc Overview} and the Cylc User Guide.}
 
 Cylc (``silk'') is a workflow engine that can manage continuous workflows of
-cycling (repeating) tasks for applications such as weather and environmental
+cycling asks for applications such as weather and environmental
```
Comment (oliver-sanders) [inline cylc-overview.tex]: *tasks

File: intro-section-apx.tex [hunk 2]
```diff
@@ -57,9 +57,10 @@ \subsection{Suite Daemons}
 submit jobs for many users).
 
 \lstinline=cylc run= starts a dedicated light-weight server program just for your
-workflow, running under your normal Unix user account. By default it is a Unix
-{\em daemon} process that detaches from your terminal and stays alive when you
-log out. To stop a suite from daemonizing use \lstinline=cylc run --no-detach=. 
+workflow, running under your normal Unix user account and it submits jobs just
```
Comment (oliver-sanders) [inline intro-section-apx.tex]: ... Unix user account ~~and~~, it submits...

**Comment by oliver-sanders** [comment]
A couple of very minor points here.

File: cylc-overview.tex (hunk 1 above)
Comment (hjoliver) [reply cylc-overview.tex]: Damn, it's amazing what mad typos my brain just skips over when I read my own text!

**Comment by hjoliver** [comment]
Thanks, fixed.

**Pull request merged**